includes/functions.php

```diff
@@ -423,18 +423,15 @@ function tsml_format_name($name, $types=null) {
 	if (!is_array($types)) $types = array();
 	if (empty($tsml_programs[$tsml_program]['flags']) || !is_array($tsml_programs[$tsml_program]['flags'])) return $name;
 	$append = array();
-	$is_meeting_online = in_array("ONL", $types, TRUE);
+	$meeting_is_online = in_array("ONL", $types, TRUE);
```

Comment:
better to use single quotes

Reply:
also the `, TRUE` wouldn't do anything in this case

Reply:
Both changed